SEP-9: Add bank branch field (#804)
stellar/stellar-protocol: ecosystem/sep-0009.md

Author: stellar.org
Status: Active
Created: 2018-07-27
Updated: 2020-12-22

> Version 1.2.1

Reviewer: As this change adds a new field, it should increment the minor number instead of the patch number.

```suggestion
Version 1.3.0
```

`bank_account_number` | string | Number identifying bank account
`bank_number` | string | Number identifying bank in national banking system (routing number in US)
`bank_phone_number` | string | Phone number with country code for bank

> `bank_branch` | string | Number identifying bank branch

Reviewer: This field could be confused for a branch name, the fact it's a number is not clear from the name. Suggest using the format off some other fields here and using the `_number` suffix.

```suggestion
`bank_branch_number` | string | Number identifying bank branch
```

> `bank_branch_number` | string | Number identifying bank branch

Reviewer 2: Unless branch identifiers are _always_ numbers, I would make this field name more generic:
```suggestion
`bank_branch_identifier` | string | An identifier for the bank branch
```
If branch _numbers_ are the standard then this suggestion can be ignored.

Reviewer 3: `bank_branch_identifier` Affin Bank 
An identifier for the. Mohd Faizal Bin Julung 
typically a branch
number. Bank 205070110343

`tax_id` | string | Tax identifier of user in their country (social security number in US)

> `tax_id_name` | string | Name of the tax ID (`SSN` or `ITIN` in the US)

Reviewer 3: Affin Bank
205070110343
